src/esw/cameras/cameras.py

```diff
@@ -515,31 +518,23 @@ def _update_active_cameras(self) -> None:
         for index, pipeline in enumerate(self._pipelines):
             self._active_cameras[index] = pipeline.device_number
 
-    def _update_all_pipe_video_info(self) -> None:
+    def _update_all_pipe_video_info_and_outputs(self) -> None:

```

Comment:
Does this need a better name? Not sure what I can call it

Reply:
how much more verbose can it be lol

Reply:
i meant a shorter name

Reply:
ill just merge the pr anyway